pkg/ccl/backupccl/restore_job.go

```
@@ -1092,14 +1096,20 @@ func (r *restoreResumer) publishTables(ctx context.Context) error {
 				return err
 			}
 			newSchemaChangeJobs = append(newSchemaChangeJobs, newJobs...)
-			existingDescVal, err := sqlbase.ConditionalGetTableDescFromTxn(ctx, txn, tbl)
+
+			existingDescVal, err := sqlbase.GetTableDescFromTxn(ctx, txn, tbl)
 			if err != nil {
-				return errors.Wrapf(err, "validating table descriptor has not changed, expected: %v", tbl)
+				log.Warningf(ctx, "restore: failed to get table desc from kv for table %d: %s",
+					tableDesc.ID, err.Error())
```

I don't know if we should be ignoring this error...